staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -515,7 +523,16 @@ func (o *AuditLogOptions) getWriter() io.Writer {
 			MaxSize:    o.MaxSize,
 		}
 	}
-	return w
+	return w, nil
+}
+
+func (o *AuditLogOptions) ensureLogFile() error {
+	mode := os.FileMode(0600)
+	f, err := os.OpenFile(o.Path, os.O_CREATE|os.O_RDWR, mode)
```

Comment:
Does this need `os.O_APPEND` to avoid clearing the file?

Reply:
Right! I had missed this. Thanks